state_machine/unit_tests/test_orbital_mechanics.py

```diff
@@ -0,0 +1,23 @@
+import unittest
+import sys
+import math
+from numpy import array
+from numpy.linalg import norm
+
+sys.path.insert(0, './state_machine/applications/flight')
+
+
+from lib.orbital_mechanics import rk4, propogate
+
+class TestRK4(unittest.TestCase):
+
+    def test_basic_difeqs(self):
+        self.assertAlmostEqual(rk4(0, 0.1, math.exp), 0.105360515657833149)
+        self.assertAlmostEqual(rk4(1, 0.1, lambda x: 3 * x), 1.349859, places=4)
+
+    def test_orbit_doesnt_crash(self):
+        """Tests if this probably stable orbit doesn't crash into the earth"""
+        x = array([0, 0, 10000.0, 5, -5, 0])
```

Comment:
It's actually fairly difficult to write down the cartesian states for a stable orbit. Might be worth double checking with orbital elements.

Reply:
Yeah, this might not be stable, this was just my attempt at a sanity check. Not too sure how to test this well. What's orbital elements?